### DISCUSSION TITLE
Need help with OSX build

### DISCUSSION THREAD

**Discussion by Godlike-Productions** [RomanKubiak/ctrlr#210] [Q&A]
Anyone familiar with building ctrlr on OSX?

I'm trying to wrap a AU binary in iLok's copy protection wrapper and am getting this error.

```
wraptool Error: pace::eden::thrift::exception::PaceThriftExceptionWire: MachOException::InsufficientHeaderPad, 12, Insufficient size available in the Mach-O header for the operation. Please try again after rebuilding the binary with the following header pad setting:
OTHER_LDFLAGS = -Xlinker -headerpad -Xlinker 800, MachOHeader.cpp, line 2780, function: pace::macho::MachOSegmentCommand *pace::macho::MachOHeader::AddSegment(const std::string &, const PaceUIntMax_T, const void *, const PaceUIntMax_T, const vm_prot_t, const vm_prot_t, const uint32_t, pace::macho::MachOSectionContainer *, uint32_t *), The desired new command size is 152, but only 120 bytes are available in the header.
```

Is there anyone who would be able to guide me through this with the required header space?

**Comment by Godlike-Productions** [comment]
This shows where to include the flags in projucer.  I can't get the code to build though in XCode.

https://forum.juce.com/t/how-to-define-other-ldflags-in-projucer-project/44836/5

**Reply by damiensellier** [reply]
for future request, the following line needs to be added in the Extra linker flags of projucer :

`-Xlinker -headerpad -Xlinker 800`


![Pace iLok linker flag for xcode](https://user-images.githubusercontent.com/55164385/155073673-3f290448-00be-41c5-9f7a-18f3c16d4d32.png)
